lib/matplotlib/tests/test_axes.py

```diff
@@ -6856,3 +6858,15 @@ def test_ylabel_ha_with_position(ha):
     ax.set_ylabel("test", y=1, ha=ha)
     ax.yaxis.set_label_position("right")
     assert ax.yaxis.get_label().get_ha() == ha
+
+
+def test_patch_bounds():
+    fig, ax = plt.subplots()
+    tol = 16*sys.float_info.epsilon
+    ax.add_patch(mpatches.Wedge((0, -1), 1.05, 60, 120, 0.1))
+    bounds = ax.dataLim.bounds
+    bot = 1.9*math.sin(15*math.pi/180)**2
```

Comment:
Just use numpy for these rather than importing `math`?  You may also want to look at `np.testing.assert_allclose` rather than writing your own tolerance.